modules/server/src/main/ts/bedrock/auto/DriverLoader.ts

```diff
@@ -56,24 +70,29 @@ const loadPhantomJs = () => {
 };
 
 export const loadDriver = (browserName: string): DriverAPI => {
-  const driverDep = browserModules[browserName];
-  if (driverDep === undefined) {
+  const driverDeps = browserModules[browserName] || [];
```

Comment:
How much do we care about our regular tinymce coding style here in bedrock? I'd be inclined to suggest something like `Obj.getOpt(browserModules, browserName).getOr([])` but I don't know if that's appropriate here

Reply:
We can't use any modules from the monorepo here, so we can't use `Optional`, `Obj`, etc... so we have to restrict the style a bit here.